hadoop-ozone/dist/src/main/compose/ozone-ha/docker-config

```diff
@@ -47,6 +48,7 @@ OZONE-SITE.XML_ozone.recon.db.dir=/data/metadata/recon
 OZONE-SITE.XML_ozone.recon.address=recon:9891
 OZONE-SITE.XML_ozone.recon.http-address=0.0.0.0:9888
 OZONE-SITE.XML_ozone.recon.https-address=0.0.0.0:9889
+OZONE-SITE.XML_dfs.container.ratis.datastream.enabled=true
```

Comment:
It this needed for this PR?

Reply:
Yes, Ozone FS tests have a test case for write with streaming, which would fail without this.

https://github.com/apache/ozone/blob/030cf40fb1252f65de238e50ee13e6c0be58ef7e/hadoop-ozone/dist/src/main/smoketest/ozonefs/ozonefs.robot#L73-L77

Reply:
That looks like a specific invocation to enable streaming vs. Setting it for all ozone ha tests.

Reply:
These are two separate settings.  `dfs.container.ratis.datastream.enabled=true` enables it at the server-side, but it's up to the client to decide if it uses regular or streaming path.  For FS API it is achieved by setting `ozone.fs.datastream.enabled`.  ([docs](https://ci-hadoop.apache.org/view/Hadoop%20Ozone/job/ozone-doc-master/lastSuccessfulBuild/artifact/hadoop-hdds/docs/public/feature/streaming-write-pipeline.html))

In short, if we want to run `ozonefs.robot` in some compose environment, we need to add this config.  It's just copied from the `ozone` and `ozonesecure` environments to the HA ones.

Reply:
BTW, you are right, the config change is not needed for _this_ PR, rather the one this is based on (see description).  Now that the base PR is merged, I've updated this one, too, and now the change is reduced.  Sorry for the confusion.